lib/decred/cw_decred.dart

```diff
@@ -2,23 +2,15 @@ part of 'decred.dart';
 
 class CWDecred extends Decred {
 	@override
-	TransactionPriority getMediumTransactionPriority() => DecredTransactionPriority.medium;	
+	TransactionPriority getMediumTransactionPriority() => DecredTransactionPriority.medium;
 
 	@override
 	WalletCredentials createDecredRestoreWalletFromSeedCredentials({
     required String name,
     required String mnemonic,
     required String password})
```

Comment:
This file doesn't look formatted.

Reply:
Yeah formatting kind of all over the place. We just want two space indention? I attempted to correct this file and everywhere that's not copied code should be ok.